CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ add_definitions(-DOPENRCT2_COMMIT_SHA1_SHORT="${OPENRCT2_COMMIT_SHA1_SHORT}")
 
 INCLUDE(FindPkgConfig)
 
+if (APPLE)
+	set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/openssl/lib/pkgconfig")
+endif (APPLE)
```

Comment:
@AaronVanGeffen If by "brewn universally", you meant force-linked (aka `brew link --force openssl`), then this avoids having to do that. It's a terrible idea to require a library to be force linked (there's a reason it isn't linked by default), and this more closely follows the solution suggested by brew itself.